modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -251,7 +251,13 @@ image:configuration/server/anti-virus-message-socket-connection-problem.png[Conf
 
 === What To Do With Infected Files Found During Scans
 
-When infected files are detected during a scan, the app can take one of two actions:
+During an upload these actions are taken:
+
+* The upload is blocked.
+* The event is logged in the owncloud server log
+* The event is reported and/or logged by the client / Web UI.
```

Comment:
I made this a separate point because, for example, the web UI reports the "infected file" response but it does not "log" it anywhere to a "permanent" file. Depending on the client, it might or might not either report, or log, or both.